helpers/src/deploy.js

```diff
@@ -52,6 +52,13 @@ async function deployRelayHub (web3, options = {}) {
     gasPrice: 1e9
   })
   console.log('paymaster ', pmInstance.options.address)
+  const forwarder = getForwarder(web3)
+  const fInstance = await forwarder.deploy({}).send({
+    from: options.from,
+    gas: 1e8,
```

Comment:
That won't work on testnet, can you reduce the gas to 1e6? we can actually have this options for all the txs as they are identical.

Reply:
And there's no need to save the contract addres for dapps to use? 
I mean `saveContractToFile(fInstance, options.workdir, 'Forwarder.json')